packages/inference/src/snippets/python.ts

```diff
@@ -237,34 +263,44 @@ export function getPythonInferenceSnippet(
 	providerModelId?: string,
 	opts?: Record<string, unknown>
 ): InferenceSnippet[] {
-	const snippets = model.tags.includes("conversational")
-		? snippetConversational(model, accessToken, provider, providerModelId, opts)
-		: model.pipeline_tag && model.pipeline_tag in pythonSnippets
-		  ? pythonSnippets[model.pipeline_tag]?.(model, accessToken, provider, providerModelId) ?? []
-		  : [];
-
-	return snippets.map((snippet) => addImportsToSnippet(snippet, model, accessToken, provider));
+	return model.pipeline_tag && model.pipeline_tag in pythonSnippets
+		? pythonSnippets[model.pipeline_tag]?.(model, accessToken, provider, providerModelId, opts) ?? []
+		: [];
 }
 
-const addImportsToSnippet = (
-	snippet: InferenceSnippet,
-	model: ModelDataMinimal,
-	accessToken: string,
-	provider: InferenceProvider
-): InferenceSnippet => {
-	let importSection: string | undefined = undefined;
-	if (snippet.client === "huggingface_hub") {
-		importSection = snippetImportInferenceClient({ accessToken, provider });
-	} else if (snippet.content.includes("requests")) {
-		importSection = snippetImportRequests({
-			accessToken,
-			model: model,
-			provider,
-			importBase64: snippet.content.includes("base64"),
-		});
+function formatBody(obj: object, format: "python" | "json" | "js" | "curl"): string {
+	if (format === "python") {
+		return Object.entries(obj)
+			.map(([key, value]) => {
+				const formattedValue = JSON.stringify(value, null, 4).replace(/"/g, '"');
+				return `${key}=${formattedValue},`;
+			})
+			.join("\n");
```

Comment:
tbh i was wondering if we shouldn't use prettier or ruff to format snippets at runtime... not sure.

Reply:
I thought so as well but not fully convinced. I feel that for some snippets it would condense more the code leading to slightly less readable snippets compared to what we have (maybe neglectable side effect?).

Anyway, maybe tooling for another day :smile: